### DISCUSSION TITLE
"Inverter Details" path not available

### DISCUSSION THREAD

**Discussion by tachnoraki** [macxq/foxess-ha#153] [Q&A] [answered]
New to this, just got an install, apologies if question is silly but I simply cannot see a solution.

The documentation state "foxesscloud_inverter_id in UUID that can be found on the foxesscloud in the url path on the Inverter Details"

When I go to https://www.foxesscloud.com/bus/device/inverter or https://www.foxesscloud.com/bus/plant/view I see my inverter but cannot click or get extra options, all I can do is create tags.

I don't seem to have that option, could my rights be restricted by the installer or something?

Where/how specifically do I get the "Inverter Details" option please? 

Thanks for the help in advance :)

**Comment by macxq** [comment, marked as answer]
on https://www.foxesscloud.com/bus/device/inverter you need to click view 

![Screenshot 2023-04-13 at 09 37 24](https://user-images.githubusercontent.com/2965092/231688319-d02384f1-0e68-482e-8ce3-a9e458b5b37f.png)

**Reply by tachnoraki** [reply]
Thanks ... turns out to be a browser thing, it doesnt work in Firefox, Chrome/Edge are OK.

<img width="400" alt="image" src="https://user-images.githubusercontent.com/6252585/231689724-a47eaf75-d761-4dc8-9161-9d523f0fad65.png">

<img width="447" alt="image" src="https://user-images.githubusercontent.com/6252585/231689952-ddd49bf2-2b59-4895-bc4f-b2224291002c.png">
